docs/client-side-connector-configuration.md

```diff
@@ -0,0 +1,71 @@
+
```

Comment:
maybe we could add `author` and `date` metas (@nono, any though about this practice to add metas in YAML in front of docs mardown files?)